apps/web/src/server/lib/nunjucks-filters/status-name.js

```diff
@@ -8,26 +8,35 @@ export const statusName = (key) => {
 	let documentStatusName = '';
 
 	switch (key) {
-		case 'checked':
+		case 'awaiting_upload':
+			documentStatusName = 'Awaiting upload';
+			break;
+		case 'awaiting_virus_check':
+			documentStatusName = 'Awaiting virus check';
+			break;
+		case 'failed_virus_check':
+			documentStatusName = 'Failed virus check';
+			break;
+		case 'not_user_checked':
+			documentStatusName = 'Unchecked';
+			break;
+		case 'user_checked':
 			documentStatusName = 'Checked';
 			break;
 		case 'ready_to_publish':
 			documentStatusName = 'Ready to publish';
 			break;
+		case 'published':
+			documentStatusName = 'Published';
+			break;
 		case 'do_not_publish':
 			documentStatusName = 'Do not publish';
 			break;
-		case 'awaiting_virus_check':
-			documentStatusName = 'Awaiting virus check';
-			break;
 		case 'unpublished':
 			documentStatusName = 'Unpublished';
 			break;
-		case 'failed_virus_check':
-			documentStatusName = 'Failed virus check';
-			break;
 		default:
-			documentStatusName = 'Unchecked';
+			documentStatusName = '';
```

Comment:
don't they need a default value?

Reply:
it's very difficult to provide one because it's a journey that can go back and forth and there's no real default except the initial status (waiting for upload) which would be unreal for most of the cases

Reply:
I think there was description of a default state on one of the file upload Tickets